projects/media-viewer/src/lib/annotations/services/annotation-api/annotation-api.service.ts

```diff
@@ -29,17 +30,15 @@ export class AnnotationApiService {
     return this.httpClient
       .post<AnnotationSet>(this.annotationSetsFullUrl, body, { observe: 'response' , withCredentials: true })
       .pipe(
-        map(response => response.body),
-        catchError(() => [])
```

Comment:
why remove the error handling?

Reply:
that code does not handle anything. also there should be catchError operator in effects to handle errors in the UI.

Reply:
Added back.